src/theme/book.css

```diff
@@ -156,6 +156,9 @@ table thead td {
 .content img {
   max-width: 100%;
 }
+pre > .buttons {
+  z-index: 2;
```

Comment:
Ah, right. I'll fix this once I figure out how to get `nib` to work on my machine.